master/internal/api_workspace.go

```diff
@@ -1293,15 +1304,35 @@ func (a *apiServer) DeleteWorkspace(
 		return nil, err
 	}
 
-	modelsExist, err := a.workspaceHasModels(ctx, req.Id)
```

Comment:
I think this was the only usage of workspaceHasModels? Might want to delete the function, if so.

Reply:
I'm using it again in the `checks` slice below, are we wanting to instead have it defined as a private function?

Reply:
Hahah all good, it's a good call out that I should probably make that variable name clearer + maybe add a comment